postgres-dataprovider-plugin/README.md

```diff
@@ -0,0 +1,85 @@
+    - Refer the following query for insertion in DB:
+      ```
+        CREATE TABLE certify.<table_name> (
+                 attribute_1 <type> NOT NULL,
+                 attribute_2 <type> NOT NULL,
+                 ...
+                 CONSTRAINT pk_id_code PRIMARY KEY (<identifier>)
+             );
+      ```
+
+2. The schema context containing all the required fields should be hosted in a public url.
+    - Refer this link for an existing context: [Registration Receipt Schema](https://piyush7034.github.io/my-files/registration_receipt.json)
+      Eg: https://<username>.github.io/<project_name>/<file_name>.json
+    - Also change the respective credential name:
+      ```
+         {
+             "@context": {
+                 "@version": 1.1,
+                 "@protected": true,
+                 "type": "@type",
+                 "schema": "https://schema.org/",
+                 "<credential_name>": {
+                     "@id": "https://<username>.github.io/<project_name>/<file_name>.json#<credential_name>"
+                 },
+                 <field1>: "schema:<type>"
+                 <field2>: "schema:<type>"
+                 ...
+             }
+         }
+      ```
+    - The primary_key should be a identifier that is existing in the current mock_identity_system records.
+      Eg: If "1234" is present in mock_identity table, then same identifier should be used for inserting records in the certify data tables
+    - When the authentication is done using this particular identifier then the record from certify tables can be fetched by the postgres plugin and returned as a JSON Object.
+
+3. Insert the templates in the DB with credential subject containing all the fields which must be the part of issued credential.
+    - Eg: Find the below template for reference:
+         ```
+             {
+               "@context": [
+                 "https://www.w3.org/2018/credentials/v1",
+                 "https://piyush7034.github.io/my-files/registration-receipt.json",
+                 "https://w3id.org/security/suites/ed25519-2020/v1"
+               ],
+               "issuer": "${issuer}",
+               "type": [
+                 "VerifiableCredential",
+                 "RegistrationReceiptCredential"
+               ],
+               "issuanceDate": "${validFrom}",
+               "expirationDate": "${validUntil}",
+               "credentialSubject": {
+                   "attributeName1": "${<attribute1>}",
+                   "attributeName2": "${<attribute2>}"
+                   ...
+               }
+             }
+         ```
+    - For referring the table creation and template insertion, see the sql scripts under certify_init.sql file: [certify-init](https://github.com/mosip/inji-certify/blob/develop/docker-compose/docker-compose-injistack/certify_init.sql)
+
+4. inji-config changes:
```

Comment:
this topic is the most important topic in this read me file to call out the config specific to this plugin and how to configure each of them